examples/lusid/orders/blocks/4-set-new-property-to-block.sql

```diff
@@ -0,0 +1,34 @@
+-- define target blocks
+@data = values
+  ('blockUpdateExample', "ORD-BLKTEST-BLK1"),
+  ('blockUpdateExample', "ORD-BLKTEST-BLK2"),
+  ('blockUpdateExample', "ORD-BLKTEST-BLK3");
+@block_ids = select
+  column1 as scope,
+  column2 as code
+FROM @data;
+
+
+-- Define contingent id
+@contingent_id = SELECT '888' as Contingent_Id;
+
+-- Add contingent Ids to the target blocks
+@block_ids_with_contingent_ids = SELECT 
+    b.scope,
+    b.code,
+    cid.Contingent_Id
+FROM 
+    @block_ids b
+CROSS JOIN (@contingent_id) cid;
```

Comment:
Not sure we need a cross join here.  We should be able to merge the 2 selects together to something like
  select @contigent_id, b.*
  from @block_ids bids
  inner join Lusid.Block b
   on .etc.etc
  inner

Reply:
Changed to inner join